torch_xla/csrc/runtime/env_hash.cc

```diff
@@ -93,7 +93,7 @@ torch::lazy::hash_t HashXlaEnvVars() {
   // Both XLA_FLAGS and LIBTPU_INIT_ARGS contain XLA flags which impact
   // the compilation result.
   static std::vector<std::string> flag_vars = {"XLA_FLAGS", "LIBTPU_INIT_ARGS"};
-  static std::vector<std::string> raw_vars = {"TPU_MEGACORE"};
+  static std::vector<std::string> raw_vars = {"TPU_MEGACORE", "XLA_HLO_DEBUG"};
```

Comment:
can we also include `XLA_IR_DEBUG`? This flag will add additional meta data to the HLO.

Reply:
Ah I see, I didn't think `XLA_IR_DEBUG` would touch the HLO, I'll include both env vars here.